UM/Qt/qml/UM/Settings/SettingCheckBox.qml

```diff
@@ -38,37 +103,6 @@ CheckBox
     {
         background: Item { }
         label: Label{ }
-        indicator: Rectangle
-        {
-            implicitWidth:  control.height;
-            implicitHeight: control.height;
-
-            color:
-            {
-                if(control.hovered || base.activeFocus)
-                {
-                    return itemStyle.controlHighlightColor
-                }
-                else
-                {
-                    return itemStyle.controlColor
-                }
-            }
-            border.width: itemStyle.controlBorderWidth;
-            border.color: control.hovered ? itemStyle.controlBorderHighlightColor : itemStyle.controlBorderColor;
-
-            UM.RecolorImage {
-                anchors.verticalCenter: parent.verticalCenter
-                anchors.horizontalCenter: parent.horizontalCenter
-                width: parent.width/2.5
-                height: parent.height/2.5
-                sourceSize.width: width
-                sourceSize.height: width
-                color: UM.Theme.colors.checkbox_mark
-                source: UM.Theme.icons.check
-                opacity: control.checked
-                Behavior on opacity { NumberAnimation { duration: 100; } }
-            }
-        }
+        indicator:
     }
-}
+}*/
```

Comment:
If you remove this comment, I think you'll see that git finds that a lot less code was changed than it reports now. Remove those old bits and pieces!